tests/test_modeling_flax_bert.py

```diff
@@ -24,6 +23,10 @@
 @require_flax
 @require_torch
 class FlaxBertModelTest(unittest.TestCase):
+    def assert_almost_equals(self, a: ndarray, b: ndarray, tol: float):
+        diff = (a - b).sum()
+        self.assertLessEqual(diff, tol, "Difference between torch and flax is {} (>= {})".format(diff, tol))
```

Comment:
```suggestion
        self.assertLessEqual(diff, tol, f"Difference between torch and flax is {diff} (>= {tol})")
```
Python 3.5 is dead, let's use f-strings pretty please!